src/main/java/org/assertj/core/util/Throwables.java

```diff
@@ -114,6 +116,26 @@ public static Throwable getRootCause(Throwable throwable) {
     return throwable;
   }
 
+  /**
+   * Get the stack trace from a {@link Throwable} as a {@link String}.
+   * 
+   * <p>The result of this method vary by JDK version as this method
+   * uses {@link Throwable#printStackTrace(java.io.PrintWriter)}.
+   * On JDK1.3 and earlier, the cause exception will not be shown
+   * unless the specified throwable alters printStackTrace.</p>
+   * 
+   * @param throwable the {@code Throwable} to get stack trace from.
+   * @return the stack trace as a {@link String}.
+   * 
+   * @author Daniel Zlotin
+   */
+  public static String getStackTrace(Throwable throwable) {
+    final StringWriter sw = new StringWriter();
+    final PrintWriter pw = new PrintWriter(sw, true);
+    throwable.printStackTrace(pw);
+    return sw.getBuffer().toString();
```

Comment:
Closing a StringWriter does nothing, and closing PrintWriter just delegates to StringWriter::close